tests/integration/api_v2/rpc/blocks.test.js

```diff
@@ -175,11 +175,88 @@ describe('Method get.blocks', () => {
 			const response = await getBlocks({ timestamp: `${from}:${to}` });
 			expect(response).toMap(jsonRpcEnvelopeSchema);
 			const { result } = response;
-			expect(result.data[0]).toMap(blockSchemaVersion5);
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
 			result.data.forEach((blockItem) => {
+				expect(blockItem).toMap(blockSchemaVersion5);
 				expect(blockItem.timestamp).toBeGreaterThanOrEqual(from);
 				expect(blockItem.timestamp).toBeLessThanOrEqual(to);
 			});
 		});
+
+		it('Blocks with from... timestamp -> ok', async () => {
+			const from = 0;
+			const response = await getBlocks({ timestamp: `${from}:` });
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
+			result.data.forEach((blockItem) => {
+				expect(blockItem).toMap(blockSchemaVersion5);
+				expect(blockItem.timestamp).toBeGreaterThanOrEqual(from);
+			});
+		});
+
+		it('Blocks with ...to timestamp -> ok', async () => {
+			const to = Math.floor(Date.now() / 1000);
+			const response = await getBlocks({ timestamp: `:${to}` });
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
+			result.data.forEach((blockItem) => {
+				expect(blockItem).toMap(blockSchemaVersion5);
+				expect(blockItem.timestamp).toBeLessThanOrEqual(to);
+			});
+		});
+	});
+
+	describe('is able to retireve block lists within height range', () => {
+		it('Blocks with min...max height -> ok', async () => {
+			const minHeight = 1;
+			const maxHeight = refBlock.height;
+			const response = await getBlocks({ height: `${minHeight}:${maxHeight}` });
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
+			result.data.forEach((blockItem) => {
+				expect(blockItem).toMap(blockSchemaVersion5);
+				expect(blockItem.height).toBeGreaterThanOrEqual(minHeight);
+				expect(blockItem.height).toBeLessThanOrEqual(maxHeight);
+			});
+		});
+
+		it('Blocks with min... height -> ok', async () => {
+			const minHeight = 0;
```

Comment:
Please consider using the `refBlock.height`.
```suggestion
			const minHeight = refBlock.height - 10;
```